aether-kernel/aether/kernel/api/tests/test_mapping_validation.py

```diff
@@ -20,13 +20,15 @@
 
 from aether.kernel.api import mapping_validation
 
+from . import NESTED_ARRAY_SCHEMA
+
 valid_schemas = {
+    'Nested': NESTED_ARRAY_SCHEMA,
     'Person': {
         'name': 'Person',
         'type': 'record',
         'namespace': 'test.Person',
-        'fields': [
-            {
+        'fields': [{
```

Comment:
:joy: Your linter has its own thoughts!

Reply:
I just added an auto-linter extension to my editor and it complained about the original format. I'm also not sure I agree. :frowning_man: